awd/dynamodb.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from enum import Enum
+
+import boto3
+from boto3.dynamodb.types import TypeDeserializer
+
+logger = logging.getLogger(__name__)
+
+
+class Status(Enum):
```

Comment:
This should be moved out, as it's not specific to persistence functionality/DynamoDB, but it's more about the general business logic. Ideally the dynamodb client has not notion of what the enum is, only the enum value (an integer value) is passed to low level persistence functions.

Reply:
Makes sense!